handshake/stk_generator.go

```diff
@@ -0,0 +1,72 @@
+package handshake
+
+import (
+	"bytes"
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go/crypto"
+)
+
+// An STK is a source address token
+type STK struct {
+	RemoteAddr string
+	SentTime   time.Time
+}
+
+// An STKGenerator generates STKs
+type STKGenerator struct {
```

Comment:
I don't really like the idea of returning the timestamp as a byte-slice from the `crypto` package.